test/blackbox-tests/test-cases/coq/duplicate-theory-workspace.t/run.t

```diff
@@ -0,0 +1,15 @@
+Duplicate theories in the same project should be caught by Dune:
+
+  $ cat > A/dune << EOF
+  > (coq.theory
+  >  (name foo))
+  > EOF
+
+  $ cat > B/dune << EOF
+  > (coq.theory
+  >  (name foo))
+  > EOF
+
+BUG this is not caught
```

Comment:
I guess we are lazier now, as the theories are not public, `dune build @install` won't complain.

Reply:
Oh yes let me check again.

Reply:
As you can see now, they are both being built.

Reply:
I guess that's OK as they are private theories, right?

Reply:
Oh yes, I guess the package layer restricts this from being an issue.

Reply:
Yes, they will never make it to the workspace theory db as they are not public.